src/retasc/validator/example_rule.yaml

```
@@ -0,0 +1,14 @@
+version: 1
+name: "Example Rule"
+prerequisites:
+  pp_schedule_item_name: "Release Date"
+  days_before_or_after: "5"
```

This should be int and I would also change it to a negative value to indicate it will be likely "before" the date:

```suggestion
  days_before_or_after: -7
```

---

Done